subsys/bluetooth/host/smp.c

```diff
@@ -616,7 +616,7 @@ static u8_t get_encryption_key_size(struct bt_smp *smp)
 /* Check that if a new pairing procedure with an existing bond will not lower
  * the established security level of the bond.
  */
-bool update_keys_check(struct bt_smp *smp)
+static bool update_keys_check(struct bt_smp *smp)
```

Comment:
This change seems unrelated? It's a good one though - just deserves its own commit, I think.

Reply:
Unrelated yes, Thought it small and harmless enough that it didn't need it's own.
Split it?

Reply:
Doesn't need its own PR, but I'd still try to keep the commits clean, so that the content matches the message, i.e. create a separate commit for this one.

Reply:
Moved to it's own commit.